src/Features/Core/Portable/ExternalAccess/VSTypeScript/Api/VSTypeScriptDocumentNavigationServiceWrapper.cs

```diff
@@ -18,6 +19,9 @@ public static VSTypeScriptDocumentNavigationServiceWrapper Create(Workspace work
             => new(workspace.Services.GetRequiredService<IDocumentNavigationService>());
 
         public bool TryNavigateToPosition(Workspace workspace, DocumentId documentId, int position, int virtualSpace = 0, OptionSet? options = null)
-            => _underlyingObject.TryNavigateToPosition(workspace, documentId, position, virtualSpace, options);
+            => this.TryNavigateToPosition(workspace, documentId, position, virtualSpace, options, CancellationToken.None);
+
+        public bool TryNavigateToPosition(Workspace workspace, DocumentId documentId, int position, int virtualSpace, OptionSet? options, CancellationToken cancellationToken)
+            => _underlyingObject.TryNavigateToPosition(workspace, documentId, position, virtualSpace, options, cancellationToken);
```

Comment:
@DanielRosenwasser @uniqueiniquity Nothing for you to do here. However, in the future it would be good to switch to this overload.  Once you do, we can remove the higher one.